src/PKSim.Core/Snapshots/Mappers/OriginDataMapper.cs

```diff
@@ -148,7 +148,11 @@ private double baseParameterValueFrom(Parameter snapshot, IDimension dimension,
          if (snapshot?.Value == null)
             return defaultValue;
 
-         var unit = dimension.UnitOrDefault(ModelValueFor(snapshot.Unit));
+         var unitName = ModelValueFor(snapshot.Unit);
+         if (!dimension.HasUnit(unitName))
```

Comment:
Making sure that we have a unit that we understand

Reply:
ACtually going to revert that because it should probably be done in core anyways